core/src/ca/kanoa/zombieworld/ZombieWorldGame.java

```diff
@@ -61,11 +62,11 @@ public void create() {
         perspectiveCamera = new PerspectiveCamera(60.0f, Gdx.graphics.getWidth(), Gdx.graphics.getHeight());
         perspectiveCamera.near = 0.1f;
         perspectiveCamera.far = 10000.0f;
-        perspectiveCamera.position.set(0.0f, 5.0f, -10.0f);
+        perspectiveCamera.position.set(0.0f, 50.0f, -10.0f);
         perspectiveCamera.lookAt(0, 0, 0);
         perspectiveCamera.update(true);
-        //perspectiveCamera.direction.set(0.0f, -1.0f, 0.0f);
-        //perspectiveCamera.up.set(0.0f, 0.0f, 1.0f);
+        perspectiveCamera.direction.set(0.0f, MathUtils.sinDeg(-70.0f), MathUtils.cosDeg(-70.0f));
```

Comment:
Angle of top down view should be held within a modifiable variable that can be changed on-the-fly